ci-operator/config/stolostron/acmqe-autotest/stolostron-acmqe-autotest-main__vboulos-acm-ocp4.12-lp-interop.yaml

```diff
@@ -0,0 +1,65 @@
+base_images:
```

Comment:
Add the following here under base_images. This tells the job created by this config to bring these images into the pipeline imagestream used by the job.
```
observability-core-automation:
    name: autotest
    namespace: acm-qe
    tag: observability_core_automation
```
If you leave this out it won't fail the automated checks but the ref will fail once it realizes it can't find the image you specify in your ref when it checks the pipeline imagestream

Reply:
Done